src/main/java/org/eclipse/xpanse/terraform/boot/config/TerraformVersionProvider.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.terraform.boot.config;
+
+import org.eclipse.xpanse.terraform.boot.models.enums.Csp;
+import org.eclipse.xpanse.terraform.boot.models.exceptions.TerraformExecutorException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+/**
+ * Version configuration classes for Terraform cloud providers.
+ */
+@Configuration
```

Comment:
we do not need this. All files will be generated by the xpanse runtime. It is completely transparent to terraform-boot.